pkg/ccl/cliccl/load_test.go

```diff
@@ -207,3 +195,24 @@ func checkExpectedOutput(t *testing.T, expected string, out string) {
 	output := out[endOfCmd+1:]
 	require.Equal(t, expected, output)
 }
+
+// Generate n continuous TimeStamps with minimal interval of 10 milliseconds to be used in incremental backup tests.
+func generateContinuousTimeStamps(n int) []hlc.Timestamp {
```

Comment:
Are these timestamps continuous? Maybe a name such as `generateTimestampsWithMinInterval` would be a more descriptive name? Or maybe just `generateBackupTimestamps`?

I also think it would be useful to mention that incremental backup collections are stored in a sub-directory structure that assumes that backups are taken at least 10 milliseconds apart.

Reply:
nit: Let's also wrap the comment to be 80 chars

Reply:
Done.